builtin/providers/aws/data_source_aws_ami.go

```diff
@@ -181,7 +181,7 @@ func dataSourceAwsAmiRead(d *schema.ResourceData, meta interface{}) error {
 	nameRegex, nameRegexOk := d.GetOk("name_regex")
 	owners, ownersOk := d.GetOk("owners")
 
-	if executableUsersOk == false && filtersOk == false && nameRegexOk == false && ownersOk == false {
+	if !executableUsersOk && !filtersOk && !nameRegexOk && !ownersOk {
```

Comment:
This seems like an unrelated change. Is there a bug here? Or is it a logically equivalent change? My gut says "no" to the later, but I'm happy to be wrong